Homework6/Homework6/Models/Bill.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Homework6.Models
+{
+	public class Bill
+	{
+        public string Id { get; set; }
+        public string ShopName { get; set; }
+        public string Date { get; set; }
```

Comment:
We must use DateTime type for Date. Also, wen we use date it's a good practise to add postfix 'at', like: CreatedAt